pyrobosim/examples/demo.py

```diff
@@ -38,16 +38,28 @@ def create_world(multirobot=False):
     world.add_room(
         name="kitchen",
         footprint=r1coords,
-        color=[1, 0, 0],
+        color="red",
         nav_poses=[Pose(x=0.75, y=0.75, z=0.0, yaw=0.0)],
     )
     r2coords = [(1.75, 2.5), (3.5, 2.5), (3.5, 4), (1.75, 4)]
-    world.add_room(name="bedroom", footprint=r2coords, color=[0, 0.6, 0])
+    world.add_room(
+        name="bedroom",
+        footprint=r2coords,
+        color="#9ACD32",  # yellowgreen
```

Comment:
I think this should also retain its original color, I don't want to change the appearance of the default example.

Didn't pick up on this before as I had not actually run your branch, sorry.

Reply:
Done.